src/hooks/useRepositoryStorage.js

```diff
@@ -39,12 +44,24 @@ function useRepositoryStorage() {
     }
   };
 
+  const removeRepo = async (target) => {
+    try {
+      const newArray = filteringArray(target);
+      await AsyncStorage.setItem(ASYNC_STORAGE_KEY, JSON.stringify(newArray));
+      notifyMessage('성공적으로 삭제되었습니다.');
+    } catch (e) {}
+  };
+
   return {
     repos: data,
     addRepo: (repos) => {
       addRepo(repos);
       return mutate();
     },
```

Comment:
```suggestion
    addRepo(repos) {
      addRepo(repos);
      return mutate();
    },
```
이렇게 단축하여 작성해도 좋을 듯 합니다. [참고](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Functions/Method_definitions)

Reply:
자료 참고하겠습니다 감사합니다!